models/DataObject/Objectbrick.php

```diff
@@ -25,6 +25,9 @@
  */
 class Objectbrick extends Model\AbstractModel
```

Comment:
```implements DirtyIndicatorAwareInterface``` maybe? and DirtyIndicatorTrait implements it then.

Reply:
@weisswurstkanone @dpfaffenbauer makes sense, we should do that